lib/worker_threads.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const {
+  drainMessagePort,
```

Comment:
Nit: FWIW, would it be better if we require `drainMessagePort` from `internal/worker/io` directly instead of `internal/worker` (then doesn't need to export in `internal/worker`), since it was exported from  `internal/worker/io` originally.

https://github.com/nodejs/node/blob/d5a5b99eafe157080db13f8221ecb64c3a092788/lib/internal/worker/io.js#L243-L244